source/_drafts/videojs-contrib-ads-6.md

```diff
@@ -0,0 +1,41 @@
+---
+title: videojs-contrib-ads 6
+author:
+  name: Greg Smith
+  github: incompl
+tags:
+---
+
+A major update to videojs-contrib-ads has been released. In this post, we'll take a look at what videojs-contrib-ads offers and what has improved in this latest version.
+
+## What is contrib-ads?
+
+Videojs-contrib-ads is a framework for creating Video.js ad plugins. Seamlessly integrating ad support into a video player can be a daunting task, especially if you have other plugins that may be effected. For example, contrib-ads has a feature called [Redispatch](https://github.com/videojs/videojs-contrib-ads#redispatch) that makes sure [media events](https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Media_events) are not impacted by the presence of ads. You may have some JavaScript on your site, or another Video.js plugin, that responds to events such as the `play` event. With contrib-ads, if an ad plays, it will not send an additional `play` event. This keeps things simple as possible without breaking other plugins. [Additional benefits of using contrib-ads are listed in the readme](https://github.com/videojs/videojs-contrib-ads#benefits).
```

Comment:
I would rephrase the redispatch part by explaining the problem first, then mentioning the contrib-ads feature.

Reply:
Made a change, how does that look?